CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 ### Grafana Mimir
 
 * [BUGFIX] Distributor: Use a boolean to track changes while merging the ReplicaDesc components, rather than comparing the objects directly. #10185
-
+* [CHANGE] Ruler: cap the rate of retries for remote query evaluation to 170/sec. This is configurable via `-ruler.query-frontend.max-retries-rate`. #10375
```

Comment:
I think the changelog in r321 was messed by: it has entries from the `2.15-rc.0` above the "Grafana Mimir". Do you think we can fix that in this PR? 🙃

Reply:
not this pr but i can open another one